spec/rails_helper.rb

```diff
@@ -34,7 +34,7 @@
 end
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = Rails.root.join("spec/fixtures")
+  config.fixture_paths = Rails.root.join("spec/fixtures")
```

Comment:
does this not require a [] or <<

```
config.fixture_paths << Rails.root.join("spec/fixtures")
OR
config.fixture_paths = [Rails.root.join("spec/fixtures")]
```

see [fixture_paths search](https://github.com/search?q=org%3Arspec%20fixture_paths&type=code)